docs/_data/team_members.yml

```diff
@@ -128,6 +128,14 @@
   number_educ: 1
   education1: Ph.D. in Communication, UC Santa Barbara
 
+- name: Tek Raj Chhetri
+  photo: ../images/tekraj.jpeg
+  info: Postdoctoral Associate
+  email: tekraj@mit.edu
+  number_educ: 2
+  education1: Ph.D. in Computer Science, University of Innsbruck
+  education2: M.Sc. in Computer Science, University of Tartu 
+
```

Comment:
please add at the end of the list.

Reply:
I have moved the description at the end of the list.